### ISSUE TITLE
unparse_content_stream broken?

### ISSUE THREAD

**Issue by mara004** [pikepdf/pikepdf#229]
Since I updated to 2.16.0, my tests involving PDFStitcher LayerFilter and the isolation gown pattern fail with the following error:
```
Traceback (most recent call last):
  File "/home/manuel/projects/pdfnodegraph/pdfstitcher/layerfilter.py", line 400, in filter_content
    page_stream = pikepdf.unparse_content_stream(commands)
  File "/home/manuel/.local/lib/python3.8/site-packages/pikepdf/models/__init__.py", line 126, in unparse_content_stream
    return _qpdf._unparse_content_stream(instructions)
SystemError: ../Objects/tupleobject.c:153: bad argument to internal function
couldn't open stream
```
I suspect this is a bug with pikepdf since no API-breaking changes should have happened?

**Comment by jbarlow83** [comment]
In a few places in layerfilter.py e.g.
`commands.append([operands,operator])`, a list is added for the operands-operator pair. The documentation calls for a tuple in that position. But the previous implementation didn't enforce that in any way, so I relaxed it to accept lists. Fixed for v2.16.1.

**Comment by mara004** [comment]
Ok, thanks. I'll patch layerfilter to use tuples soon.
